rts/Game/Game.cpp

```diff
@@ -1900,6 +1903,7 @@ void CGame::Reload()
 		// This reloads heightmap, triggers Load call-in, etc.
 		// Inside the Load call-in, Lua can ensure old units are wiped before new ones are placed.
 		saveFileHandler->LoadGame();
+		saveFileHandler->LoadAIData();
```

Comment:
I have no idea why the `reload` function even exists, I have a suspicion nothing calls it and it can be deleted.
The code is also buggy

Reply:
Called in ActionExecutor (maybe new in BAR105)
https://github.com/beyond-all-reason/spring/blob/4f05280a7f2e717833b2ec08b05346e37c4667f5/rts/Game/UnsyncedGameCommands.cpp#L3294

Reply:
It's insanely old, should be deleted.